examples/framework/preferences/minimalEditor/schemas/primary.js

```diff
@@ -0,0 +1,37 @@
+ /*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+   /**
+     * Primary Schema
+     * This schema defines the preference(s) edited by this preferences editor:
+     * their names, types, default values, etc.
+     */
+    fluid.defaults("minEditor.primarySchema", {
```

Comment:
@jobara,  it sounds to me like having each preference in its own schema (i.e. using its own fluid.defaults() call, the way our own StarterSchemas work) is preferred (@amb26, if I'm reading this wrong, let me know). I will make that change. I'll also update the code (and associated tutorial text) to give an organizational namespace to things, as suggested.